notes/dims_slicing_thoughts.md

```diff
@@ -0,0 +1,162 @@
+# slicing data - thoughts
+
+## Spaces
+- rendering space (2D/3D)
+- nagivable nD space (nD)
+
+If we can separate concerns over these things and still allow for 
+reshuffling dims and computing dynamic projections then we're onto a winner
+
+## Data Model
+We need to be explicit about different kinds of data, our full nD data and 
+the visualisable 2D/3D data which will be rendered at any given moment
+
+### nD Data
+this is arbitrary nD array data
+`data: ArrayLike[...]`
+
+This data will be transformed into one of the following for slicing
```

Comment:
"for" slicing? or "by" slicing?

maybe a really rudimentary diagram here?   Something like this (though, I suspect this **wrong**)

```
nD Data --[slicing]-->  ...  --[reduction]--> Renderable Data
```